libraries/AR_WPNav/AR_WPNav.h

```diff
@@ -0,0 +1,144 @@
+#pragma once
+
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Common/AP_Common.h>
+#include <APM_Control/AR_AttitudeControl.h>
+#include <AP_Navigation/AP_Navigation.h>
+
+#define AR_WPNAV_HEADING_UNKNOWN    99999.0f    // used to indicate to set_desired_location method that next leg's heading is unknown
```

Comment:
I'd much prefer to see this as a `const float` rather than a `#define`...

Reply:
ok, changed to a const float!